File: app/src/main/java/com/BlizzardArmory/warcraft/Talents/Talent.java

package com.BlizzardArmory.warcraft.Talents;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Talent {

    @SerializedName("talent")
    @Expose
    private TalentName talent;
    @SerializedName("spell_tooltip")
    @Expose
    private SpellTooltip spellTooltip;

    public TalentName getTalent() {
        return talent;
    }

    public void setTalent(TalentName talent) {
        this.talent = talent;
    }

    public SpellTooltip getSpellTooltip() {
        return spellTooltip;
    }

    public void setSpellTooltip(SpellTooltip spellTooltip) {
        this.spellTooltip = spellTooltip;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/talents/Talent.kt
package com.BlizzardArmory.model.warcraft.talents

import com.google.gson.annotations.SerializedName


/**
 * The type Talent.
 */
data class Talent(

    @SerializedName("talent")
    var talent: TalentName,

    @SerializedName("spell_tooltip")
    var spellTooltip: SpellTooltip

)